apps/site/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,77 @@
+"use client";
+
+import Image from "next/image";
+
+import styles from "./Footer.module.scss";
+
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+import Mail from "@/assets/icons/mail.svg";
+import Facebook from "@/assets/icons/facebook.svg";
+import Instagram from "@/assets/icons/insta.svg";
+import Twitter from "@/assets/icons/twitter.svg";
+
+type Social = {
+	icon: any;
+	link: string;
+	alt: string;
+	width: number;
+	height: number;
+};
+
+const SOCIALS: Social[] = [
+	{
+		icon: Mail,
+		link: "mailto:hack@uci.edu",
+		alt: "Mail",
+		width: 79,
+		height: 59,
+	},
+	{
+		icon: Facebook,
+		link: "https://www.facebook.com/UCI.Hack/",
+		alt: "Facebook",
+		width: 59,
+		height: 59,
+	},
+	{
+		icon: Instagram,
+		link: "https://www.instagram.com/hackatuci/",
+		alt: "Instagram",
+		width: 69,
+		height: 72,
+	},
+	{
+		icon: Twitter,
+		link: "https://www.twitter.com/HackUCI",
+		alt: "Twitter",
+		width: 64,
+		height: 53,
+	},
+];
+
+export default function Footer() {
+	return (
+		<footer className={styles.footer}>
+			<p className={styles.text}>Made with &lt;3 in Irvine, CA - Hack @ UCI</p>
+			<Row
+				className="justify-content-center align-items-center"
+				xs="auto"
+				sm="auto"
+				md="auto"
+				lg="auto"
+				xl="auto"
+				xxl="auto"
+			>
+				{SOCIALS.map(({ icon, link, alt, width, height }: Social) => (
+					<Col>
+						<a key={link} href={link} target="_blank">
+							<Image src={icon.src} alt={alt} width={width} height={height} />
+						</a>
+					</Col>
+				))}
+			</Row>
```

Comment:
There's a slight horizontal scroll when viewing the site. This should be fixed by wrapping the `Row` with `Container`.

Reply:
This entire approach seems a lot more complicated than it needs to be: what was wrong with the [original Footer design from 927780c](https://github.com/HackAtUCI/zothacks-site-2023/pull/14/commits/927780c557216cecc02f80fda54a39a88d1ac02a#diff-a9503fb779eb684b10a1f1537c8a2f59b987c76df1145ada0a98461d23a7e00c)?